src/store/reducers/__tests__/modal.spec.js

```diff
@@ -0,0 +1,36 @@
+import store from 'store/store';
+import reducer from '../modal';
+import { openModal, closeModal } from 'store/actions';
+import { modalType } from 'components/Modal/Modal';
+
+describe('Modal reducer', () => {
+  const movieId = '1';
+
+  it('returns the initial state', () => {
+    expect(reducer(undefined, {})).toMatchSnapshot();
+  });
```

Comment:
I believe there is no reason to use snapshot tests on reducers. Snapshots should be used for eventual html to see, but with objects you can use simple https://jestjs.io/docs/jest-object#jestresetallmocks